### ISSUE TITLE
Epoch needs to be updated in RSS Kopernicus config

### ISSUE THREAD

**Issue by Arrowstar** [KSPtoMars/KSPtoMarsTools#20]
In .\GameData\RealSolarSystem\RSSKopernicus.cfg

Change the Epoch line from:

Epoch = -31542641.784 

TO:

Epoch = -2524614641.784

This sets the game UT=0 epoch at January 1, 2030.

THIS IS A CRITICAL CHANGE!  WE WILL NOT GO TO MARS WITHOUT THIS CHANGE.  ALL MISSIONS HAVE BEEN PLANNED WITH THIS CHANGE IN MIND.

**Comment by Arrowstar** [comment]
By the way, when you go to make this change, you'll find that KSP no longer lets you get past the main menu (load game button no longer works).  To bypass this:

1) Start a game using the default Epoch line in the CFG file, the one that ships with the file originally.

2) Install Dev Helper mod: http://forum.kerbalspaceprogram.com/threads/60880-0-24-Magic-Smoke-Industries-DevHelper-v0-6

3) Change the Epoch line to what I specify in the original issue post.

4) Start KSP and have Dev Helper load directly to the VAB in the game you created in (1).  Exit VAB with button in upper right and everything should work normally.

If this works, and it works on my PC, Dev Helper may need to get added to the modpack.

**Comment by Arrowstar** [comment]
And I can confirm that what I wrote above in the previous post works.

**Comment by frenzeldk** [comment]
@Arrowstar is it necessary to start the game once with the original epoch?

**Comment by Arrowstar** [comment]
Yes, you need to start a game and end up at the space center before it will
work. Need to give Dev Helper a game to launch into later.
